### PR TITLE
Translation for de-DE

### PR DESCRIPTION
**Pull request by Fabian9799**
Added Translation for German

### CODE REVIEW

File: src/assets/locales/de/DE/translation.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    "commonTasks": "Allgemeine Aufgaben",
+    "wiringTasks": "Verkabelungsaufgaben",
+    "ventConnections": "Lüftungsanschlüsse",
+    "securityCameras": "Sicherheitskameras",
+    "cameraZones": "Kamera Sichtbereiche",
+    "reactor": "Reaktor",
+    "upperEngine": "Oberer Antrieb",
+    "lowerEngine": "Unterer Antrieb",
+    "security": "Überwachung",
+    "medBay": "MedBer",
```
Comment (I3lackRacer) [inline src/assets/locales/de/DE/translation.json:41]: 'Krankenstation'

File: src/assets/locales/de/DE/translation.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    "resetColors": "Farben zurücksetzen",
+    "change": "Ändern",
+    "backgroundColor": "Hintergrundfarbe",
+    "textColor": "Textfarbe",
+    "inputBackgroundColor": "Hintergrundfarbe eingeben",
+    "inputTextColor": "Textfarbe eingeben",
+    "neutralBackgroundColor": "Neutrale Hintergrundfarbe",
+    "neutralTextColor": "Neutrale Textfarbe",
+    "innocentBackgroundColor": "Hintergrundfarbe für Unschuldige",
+    "innocentTextColor": "Textfarbe für unschuldige",
```
Comment (I3lackRacer) [inline src/assets/locales/de/DE/translation.json:85]: 'Unschuldige' capitalization

**Review by I3lackRacer — CHANGES_REQUESTED** [review]

**Comment by Kedyn** [comment]
Thank you for your contribution, we are currently in the process of another big update to the UI, would you mind holding off for a bit until we get that done. Thank you in advance.

**Comment by Kedyn** [comment]
@Fabian9799 I think we are done with all the new UI things we wanted to translate, so you are welcome to translate the new things as shown: https://github.com/Kedyn/fusliez-notes/tree/master/src/assets/locales

**Comment by Fabian9799** [comment]
Created newer version: [https://github.com/Kedyn/fusliez-notes/pull/67](https://github.com/Kedyn/fusliez-notes/pull/67)

**Pull request closed without merging**